src/RemoteRequest/CurlRemoteGetRequest.php

```diff
@@ -97,13 +98,17 @@ public function __construct($sslVerify = true, $timeout = self::DEFAULT_TIMEOUT,
     public function get($url, $headers = [])
     {
         $retriesLeft = $this->retries;
+
+        if (! is_string($url) || empty($url)) {
+            throw FailedToGetFromRemoteUrl::withException($url, FailedToParseUrl::forUrl($url));
+        }
+
         do {
             $curlHandle = curl_init();
 
             curl_setopt($curlHandle, CURLOPT_URL, $url);
-            curl_setopt($curlHandle, CURLOPT_HEADER, 0);
             curl_setopt($curlHandle, CURLOPT_FOLLOWLOCATION, true);
-            curl_setopt($curlHandle, CURLOPT_SSL_VERIFYPEER, $this->sslVerify ? 1 : 0);
+            curl_setopt($curlHandle, CURLOPT_SSL_VERIFYPEER, $this->sslVerify ? true : false);
```

Comment:
Seems this could now just be simply:

```suggestion
            curl_setopt($curlHandle, CURLOPT_SSL_VERIFYPEER, $this->sslVerify);
```